example.js

```diff
@@ -137,4 +141,25 @@ shippo.customsdeclaration.create({
 		//Deal with an error with the transaction
 		console.log("Message: %s", JSON.stringify(transaction.messages, null, 2));
 	}
+	var requested_start_time = new Date();
+	requested_start_time.setDate(requested_start_time.getDate() + 1);
+	var requested_end_time = new Date();
+	requested_end_time.setDate(requested_start_time.getDate() + 1.1);
```

Comment:
what does adding 1.1 to a Date do?

Reply:
This gives a window for a pick up:
   start: tomorrow at current time -> tomorrow @ 12:00
   end: tomorrow at current time + 0.1 days (2.4 hrs) -> tomorrow @ 14:24